src/errors/convert.js

```diff
@@ -24,13 +24,17 @@ import MailboxNotCachedError from './MailboxNotCachedError'
 import NoDraftsMailboxConfiguredError from './NoDraftsMailboxConfiguredError'
 import NoSentMailboxConfiguredError from './NoSentMailboxConfiguredError'
 import NoTrashMailboxConfiguredError from './NoTrashMailboxConfiguredError'
+import CouldNotConnectError from './CouldNotConnectError'
+import ManageSieveError from './ManageSieveError'
 
 const map = {
 	'OCA\\Mail\\Exception\\DraftsMailboxNotSetException': NoDraftsMailboxConfiguredError,
 	'OCA\\Mail\\Exception\\MailboxLockedException': MailboxLockedError,
 	'OCA\\Mail\\Exception\\MailboxNotCachedException': MailboxNotCachedError,
 	'OCA\\Mail\\Exception\\SentMailboxNotSetException': NoSentMailboxConfiguredError,
 	'OCA\\Mail\\Exception\\TrashMailboxNotSetException': NoTrashMailboxConfiguredError,
+	'OCA\\Mail\\Exception\\CouldNotConnectException': CouldNotConnectError,
+	'Horde\\ManageSieve\\Exception': ManageSieveError,
```

Comment:
@ChristophWurst should we wrap `Horde\ManageSieve\Exception`?

Reply:
Using the one from horde is also fine by me